woocommerce/woocommerce-ios
# Login: Native Jetpack setup UI

Part of #8075


### Description

This PR adds UI for the native Jetpack setup screen and handles the navigation from the site credential login screen to this screen.

Changes include:
- Updated `SiteCredentialLoginViewModel` to prepare cookie authentication with the logged in credentials and trigger a callback to the coordinator.
- A new view and view model for the setup steps - the actual implementation of the steps will be handled in a subsequent PR.
- Updated `LoginJetpackSetupCoordinator` to navigate to `LoginJetpackSetupView` after login succeeds.

### Testing instructions


- Log out of the app or skip login onboarding if needed.
- On the prologue screen, select "Log In" or "Continue with WordPress.com" based on the A/B test variant you get.
- Log in with your WordPress.com account.
- If your account doesn't have any associated site, notice that the empty store picker is displayed with the matching design as on Figma.
- Select Add a Store > Connect an existing site.
- Enter the address of your test store and tap Continue.
- On the Jetpack error screen, select Install Jetpack or Connect Jetpack.
- When the site credential login screen is presented, enter the correct credentials for your store.
- Notice that a new screen for the setup steps is presented. The screen should match the design in ERGtCFLyiC9CFpHsw1XIid-fi-1110%3A37319 and reflects the correct state of the site's Jetpack.
- Feel free to update the current setup/connection steps in `LoginJetpackSetupViewModel` and see if the UI is still correct.

### Screenshots

| Install & connect | Connection only | 
| ----- | ----- |
| <img src="https://user-images.githubusercontent.com/5533851/202661679-3754ccbc-a135-4350-91c1-eb5cea0c3153.png" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/202660902-d91a98c0-7953-4479-97ac-103452afad89.png" width=320 /> |

---
